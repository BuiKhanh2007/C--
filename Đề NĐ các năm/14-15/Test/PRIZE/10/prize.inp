14432664233136161134331155531242562355614145565351313424416341322612155443641231556436222152155121421311623343463516145553444512625215144415243264562454545244356451643432515514525365452626545525115315
4152245165334322662544645524561241464362545561635154116655465555434243236453145666523434451233541431135536114164566431256253434534