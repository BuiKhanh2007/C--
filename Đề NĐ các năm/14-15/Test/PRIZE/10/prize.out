14432664233136161134331155531242562355614145565351313424416341322612159595886396890758884696800645982553087706009077780707561168090770578658479717708121068678807685184863651050639530019057801778549849
9